polling_stations/apps/data_importers/management/commands/import_solihull.py

```diff
@@ -3,91 +3,79 @@
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
-    council_id = "E08000029"
-    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.CSV"
-    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.CSV"
-    elections = ["parl.2019-12-12"]
-    allow_station_point_from_postcode = False
+    council_id = "SOL"
+    addresses_name = "2021-04-07T11:17:55.688819/Democracy_Club__06May2021.CSV"
+    stations_name = "2021-04-07T11:17:55.688819/Democracy_Club__06May2021.CSV"
+    elections = ["2021-05-06"]
 
     def station_record_to_dict(self, record):
         # These polling places have a UPRN, and the addressbase postcode doesn't match
         # the postcode from the council. In these cases the addressbase postcode matches
         # the postcode used on the venue's website.
-        # Online references toStation ID 7680 (Whar Hall Road Community Centre) don't
+        # Online references to station ID 7680 (Whar Hall Road Community Centre) don't
```

Comment:
Can you drop the station id as it probably doesn't apply anymore.

Reply:
Yep

Reply:
I've dropped the ID and reworded for clarity.